src/Workspaces/Core/Portable/LanguageServices/SyntaxFacts/ISyntaxKindsService_LanguageService.cs

```diff
@@ -6,7 +6,7 @@
 
 using Microsoft.CodeAnalysis.Host;
 
-namespace Microsoft.CodeAnalysis.LanguageServices
+namespace Microsoft.CodeAnalysis
```

Comment:
I will revert this namespace change once https://github.com/dotnet/roslyn/pull/42019 is merged.